src/lib/y2network/sysconfig/interfaces_reader.rb

```diff
@@ -105,7 +105,7 @@ def find_connections
       # @option data [String] "name"     Device description
       # @option data [String] "type"     Device type ("eth", "wlan", etc.)
       def build_physical_interface(data)
-        Y2Network::PhysicalInterface.new(data["dev_name"]).tap do |iface|
+        Y2Network::Interfaces::PhysicalInterface.new(data["dev_name"]).tap do |iface|
```

Comment:
hmm, how this will wotk with that specialization? e.g. if we have specialized wlan one?

Reply:
I don't know and I didn't check this code in depth yet. I think it is in a prototype state now. It also is not creating virtual devices yet.